app/main.js

```diff
@@ -91,6 +92,23 @@ window.initApp = () => {
   }
 };
 
+const detection = new UnsupportedBrowserDetection();
+const checkSupportedBrowser = () => {
```

Comment:
Check means that something is being evaluated and returns `true` or `false`. This function initializes the whole react application after checking.

Maybe we could move the detection inside the `initApp` function?

Reply:
Refactored together with removing window.onload